HIRS_Utils/src/main/java/hirs/data/persist/SupplyChainValidationSummary.java

```diff
@@ -26,6 +36,8 @@
     @JoinColumn(name = "device_id")
     private final Device device;
 
+    private static final String DEVICE_ID_FIELD = "device.id";
```

Comment:
They shouldn't be the same because they really are different fields in different objects. If we ever had to make a change in one, we wouldn't necessarily want that to point to the other, so I made the other protected.